src/main/electron.ts

```diff
@@ -140,15 +141,44 @@ const initIPC = () => {
   })
 }
 
+autoUpdater.logger = {
+  ...log,
+  // Set custom info logger for better understanding in common debug-flow
+  info(message?: unknown): void {
+    console.log('[AutoUpdater.info]', message)
+  }
+}
+interface SendUpdateMessage {
+  (message: IPCMessages.UPDATE_AVAILABLE, version: string): void
+  (message: IPCMessages.UPDATE_NOT_AVAILABLE): void
+}
+const sendUpdateMessage: SendUpdateMessage = (message: IPCMessages, version?: string) => {
+  if (mainWindow) {
+    mainWindow.webContents.send(message, version)
+  }
+}
+
+// Disable autoDownload
+autoUpdater.autoDownload = false
+autoUpdater.allowPrerelease = true
+
 const init = async () => {
   await app.whenReady()
   await initMainWindow()
   app.on('window-all-closed', allClosedHandler)
   app.on('activate', activateHandler)
-
   initIPC()
+  await autoUpdater.checkForUpdates()
```

Comment:
Extract `autoUpdater.checkForUpdates()` into `main/api/appUpdate.ts` to manage requests for checking updates from `renderer` side something like this:
```
// renderer -> main
CHECK_APP_UPDATE

// renderer -> main
APP_UPDATE
```

With this, we can start requests every 6 (or 12hrs etc.) to let users know about updates, even if an user has open it's wallet for two days or so...